content/07.pathogenesis.md

```diff
@@ -135,3 +135,43 @@ The network analysis showed in particular that non-structural proteins 9 and 10
+As the viral receptor ACE2 has low expression in A549 cells, authors also supplemented A549 cells with adenovirus (AdV)-based vectors expressing mCherry (control) or ACE2 (A549-ACE2). 
+All infections were performed at a high multiplicity of infections (MOI, 2–5), with the exception of A549-ACE2 cells, which were infected with SARS-CoV-2 at both low (0.2 MOI) and high MOI (2 MOI).
+Poly(A) bulk RNA sequencing (RNA-seq) of host cells was performed at 24 hours post-infection (hpi) of all the viruses except IAV (9-12 hours).
+The authors also measured host transcriptional responses to SARS-CoV-2 in primary normal human bronchial epithelial cells (HBEC or NHBE cells), nasal washes of ferrets as an animal model, and lung samples from two COVID-19 patients. 
+Additionally, transcriptional response to IAV in NHBE cells and nasal washes of ferrets was measured.
+Differential expression (DE) analysis was carried out between infected and respective mock (control) cell condition.
+DE analysis revealed that the transcriptional response of the cells to efficient SARS-CoV-2 replication was significantly different from the host response to all of the other viruses tested. 
+A unique proinflammatory cytokine signature associated with SARS-CoV-2 was present under both high and low MOI conditions. 
+However, A549-ACE2 cells in the low-MOI condition showed no significant IFN-I or IFN-III expression, but cells at high-MOI did.
+This finding suggested that IFN induction is MOI dependent. 
+Similarly, in cells from the NHBE line, ferrets, and COVID-19 patients, they found significant enrichment of chemokine signaling but no significant induction of IFN-I or IFN-III. 
+Together, these results suggest that in contrast to common respiratory viruses, SARS-CoV-2 induces limited antiviral state with low IFN-I or IFN-III expression and a moderate IFN-stimulated genes (ISG) response. 
+However, the low IFN induction in ACE2-expressing A549 cells could be overcome by using a 10-fold increase in SARS-CoV-2 (MOI, 2), suggesting a SARS-CoV-2 antagonist that is rendered ineffective under high-MOI conditions [@doi:10.1016/j.cell.2020.04.026]. 
+This hypothesis was further supported by a recent study [@doi:10.1101/2020.05.11.088179] that showed that the SARS-CoV-2 _ORF3b_ gene suppresses IFNB1 promoter activity (IFN-I induction) more efficiently than the SARS-CoV-1 _ORF3b_ gene, which is considerably longer.
+
+A second study by Wyler et al. [@doi:10.1101/2020.05.05.079194] examined three human cell lines: H1299 (human non-small cell lung carcinoma cell line), Calu-3, and Caco-2 (human epithelial colorectal adenocarcinoma cell line). 
+They performed a comprehensive analysis of the transcriptional response to SARS-CoV-2 and SARS-CoV-1 at different hpi (4-36 hours) with an MOI of 0.3. 
+Using poly(A) bulk RNA-seq, the authors found negligible susceptibility of H1299 cells (< 0.08 viral read percentage of total reads) in contrast to Caco-2 and Calu-3 cells (>10% of viral reads). 
```

Comment:
What is different about these cell lines? What does it tell us that some are more susceptible?

Reply:
suggesting a role of cell-type specific host factors.